src/solidity-frontend/solidity_convert.cpp

```diff
@@ -2071,8 +2078,7 @@ bool solidity_convertert::get_decl_ref_builtin(
   code_typet convert_type;
   typet return_type;
   if(
-    decl["name"] == "assert" || decl["name"] == "__ESBMC_assume" ||
-    decl["name"] == "__VERIFIER_assume")
+    name == "assert" || name == "__ESBMC_assume" || name == "__VERIFIER_assume")
```

Comment:
Should ` name == "assert"` be ` name == "assume"`?

Reply:
The `name == assert` is for assert keywords in Solidity. Here “assume”(or "require") has already been converted to "__ESBMC_assume", thus we have `name == "__ESBMC_assume"`.